eventdata/challenges/large-shard-sizing.json

```diff
@@ -60,11 +60,7 @@
     },
     {
       "name": "refresh-auto-{{ p_size }}",
-      "operation": {
-        "operation-type": "raw-request",
-        "method": "POST",
-        "path": "/elasticlogs-auto/_refresh"
-      },
+      "operation": "refresh",
```

Comment:
Rally's implementation refreshes all indices by default. Is this intended? If not you can specify the name of the index with the `index` property (see [docs](https://esrally.readthedocs.io/en/stable/track.html#refresh)).

Reply:
That should be fine.